### PR TITLE
Add Proposal: Signaling proposals

### PR DESCRIPTION
**Pull request by rperez89**
Screen Shot 2020-08-04 at 6 33 47 PM
Screen Shot 2020-08-04 at 6 34 09 PM

### CODE REVIEW

**Comment by willjgriff** [comment]
Can we hide the requested amount and beneficiary fields if signalling is checked?

**Comment by rperez89** [comment]
> Can we hide the requested amount and beneficiary fields if signalling is checked instead of just disabling them?

ahhh wait there is a new ui, need to update the screenshots

File: src/components/AddProposalPanel.js [hunk 1]
```diff
@@ -156,16 +173,22 @@ const AddProposalPanel = React.memo(({ onSubmit }) => {
...
-        stake reaches above the threshold.
-      </Info>
+        <DropDown
+          header="Select proposal type"
+          placeholder="Proposal type"
+          selected={formData.proposalType}
+          onChange={handleProposalTypeChange}
+          items={['Signaling proposal', 'Funding proposal']}
+          required
+          css="width: 100%;"
```
Comment (fabriziovigevani) [inline src/components/AddProposalPanel.js:189]:
```suggestion
          wide
```

File: src/components/AddProposalPanel.js [hunk 2]
```diff
@@ -220,6 +243,50 @@ const AddProposalPanel = React.memo(({ onSubmit }) => {
...
+        <Info
+          title="Action"
+          css={`
+            margin-top: ${3 * GU}px;
+          `}
+        >
+          {fundingMode ? (
+            <>
+              <span>
+                This action will create a proposal which can be voted on{' '}
```
Comment (fabriziovigevani) [inline src/components/AddProposalPanel.js:259]:
```suggestion
                This action will create a proposal which can be voted on
```

File: src/components/AddProposalPanel.js [hunk 3]
```diff
@@ -18,6 +19,21 @@ import { calculateThreshold, getMaxConviction } from '../lib/conviction'
 
 const ZERO_ADDR = '0x0000000000000000000000000000000000000000'
 
+const NULL_PROPOSAL_TYPE = -1
+// const SIGNALING_PROPOSAL = 0
```
Comment (fabriziovigevani) [inline src/components/AddProposalPanel.js:23]: Should we remove it or uncomment it?

File: package.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
     "@1hive/1hive-ui": "^1.0.2",
     "@1hive/connect-thegraph-conviction-voting": "^0.1.7",
     "@1hive/uniswap-sdk": "^1.0.0",
-    "@aragon/connect": "^0.3.0",
+    "@aragon/connect": "^0.5.0",
```
Comment (Evalir) [inline package.json:9]: I think we should upgrade this to the [latest patch version](https://github.com/aragon/connect/releases/tag/v0.5.2). :)

Comment (fabriziovigevani) [reply package.json:9]: Ah Pierre already did that here! https://github.com/1Hive/honey-pot/pull/36

**Pull request merged**